Automattic/wp-calypso
# Assembler - Fastrack - Fix sidebar alignment issues

Related to #78513

## Proposed Changes

* Fixed multiple alignment issues on the sidebar to match the design - Figma Tv3pYqA3EcRfiXC31IxrXE-fi-751%3A31932

### Before
|Main|Header|Homepage|Add Patterns|Footer|Colors|Fonts|
|--|--|--|--|--|--|--|
|<img width="358" alt="Screenshot 2566-06-26 at 19 01 25" src="https://github.com/Automattic/wp-calypso/assets/1881481/f96a99d1-11a0-4ae0-b781-7bb8ee72651d">|<img width="358" alt="Screenshot 2566-06-26 at 19 00 20" src="https://github.com/Automattic/wp-calypso/assets/1881481/03de9f1c-fe93-408b-a3da-0ad148ab6e05">|<img width="357" alt="Screenshot 2566-06-26 at 19 01 46" src="https://github.com/Automattic/wp-calypso/assets/1881481/a290bc8e-38da-421b-aa09-c6eb46cac605">|<img width="695" alt="Screenshot 2566-06-26 at 19 01 03" src="https://github.com/Automattic/wp-calypso/assets/1881481/37ff41eb-29ee-4698-b4d4-3235c948dfd3">|<img width="359" alt="Screenshot 2566-06-26 at 19 02 02" src="https://github.com/Automattic/wp-calypso/assets/1881481/9258386d-8c65-4068-ae42-f000096af675">|<img width="358" alt="Screenshot 2566-06-26 at 19 02 16" src="https://github.com/Automattic/wp-calypso/assets/1881481/0e579661-c500-456c-87c2-7efa0d300118">|<img width="359" alt="Screenshot 2566-06-26 at 19 02 32" src="https://github.com/Automattic/wp-calypso/assets/1881481/f52402c2-686b-460d-8706-0fe32516bac4">|

### After
|Main|Header|Homepage|Add Patterns|Footer|Colors|Fonts|
|--|--|--|--|--|--|--|
|<img width="359" alt="Screenshot 2566-06-26 at 19 56 18" src="https://github.com/Automattic/wp-calypso/assets/1881481/7c96c028-7aac-4afc-a531-6e5fc439f1a3">|<img width="360" alt="Screenshot 2566-06-26 at 19 56 36" src="https://github.com/Automattic/wp-calypso/assets/1881481/e8aa083b-16c1-4c74-be22-de7fd081a21e">|<img width="359" alt="Screenshot 2566-06-26 at 19 57 29" src="https://github.com/Automattic/wp-calypso/assets/1881481/adee6da7-2702-4d2b-93c4-5a7385de138a">|<img width="696" alt="Screenshot 2566-06-26 at 19 56 53" src="https://github.com/Automattic/wp-calypso/assets/1881481/595f5030-64d0-4875-a608-a5bf24697149">|<img width="359" alt="Screenshot 2566-06-26 at 19 57 42" src="https://github.com/Automattic/wp-calypso/assets/1881481/4f18dc8d-4383-40b2-afe0-ad238c20689a">|<img width="358" alt="Screenshot 2566-06-26 at 19 57 58" src="https://github.com/Automattic/wp-calypso/assets/1881481/fa1d2e44-8bad-4692-9129-cd98c9e38a6f">|<img width="358" alt="Screenshot 2566-06-26 at 19 58 10" src="https://github.com/Automattic/wp-calypso/assets/1881481/4457d0db-27b8-4369-a891-840389b25615">|



## Testing Instructions



* Using the Calypso Live link below 
* Access the assembler `/setup/with-theme-assembler/patternAssembler?siteSlug={ YOUR SITE }&theme=blank-canvas-3`
* Verify that all alignments, especially the vertical spacings, look right on the sidebar
* Navigate with the keyboard to verify the border added on focus is visible and aligned

## Pre-merge Checklist
